File: Front/src/app/admin/perfil/usuario.dto.ts
export interface UsuarioDTO {
  id: number;
  nombreCompleto: string;
  email: string;
  telefono: string;
  documento: string;
  rol: string;
  fechaRegistro: string;
}

export interface ActualizarPerfilDTO {
  nombreCompleto?: string;
  telefono?: string;
  documento?: string;
  email?: string;
  contrasena?: string;
}

File: Back/src/main/java/com/example/Back/Dto/UsuarioDTO.java
package com.example.Back.Dto;

import lombok.Data;
import java.util.List;

@Data
public class UsuarioDTO {
    private Integer idUsuario;
    private String nombreCompleto;
    private String email;
    private String telefono;
    private String documento;
    private List<String> roles;
    private Boolean estado;
    private String fechaRegistro;
    private String fotoUrl;
}